tests/spark/autologging/ml/test_pyspark_ml_autologging.py

```diff
@@ -182,12 +181,25 @@ def load_model_by_run_id(run_id, model_dir=MODEL_DIR):
     return mlflow.spark.load_model(f"runs:/{run_id}/{model_dir}")
 
 
+def validate_logged_model_predict(model_uri):
+    mlflow_model = Model.load(model_uri)
+    input_example = mlflow_model.load_input_example()
+    pyfunc_model = mlflow.pyfunc.load_model(model_uri)
+    pyfunc_model.predict(input_example)
+
+
+def validate_logged_model_signature(model_uri, input_schema, output_schema):
+    mlflow_model = Model.load(model_uri)
+    assert mlflow_model.signature.inputs.to_dict() == input_schema
+    assert mlflow_model.signature.outputs.to_dict() == output_schema
```

Comment:
```suggestion
    assert mlflow_model.signature = signaure
```

does `Signature` support `__eq__`? If it doesn't, ignore this suggestion.